src/types.rs

```diff
@@ -258,6 +267,8 @@ pub struct FullCrate {
     pub downloads: Downloads,
     pub owners: Vec<User>,
     pub reverse_dependencies: Vec<Dependency>,
+    pub reverse_dependencies_versions: Vec<Version>,
```

Comment:
I assume the new response looks that way for caching purposes, but it's pretty confusing to use. I think the `version` and `de

We can make this nicer by adding a new type:

```rust
pub struct ReverseDependency {
  pub crate_version: Version,
  pub dependency: Dependency,
}
```

Reply:
adding an additional method that fetches a single page makes sense.

probably: 

```rust
fn crate_reverse_dependencies(&self, _crate: &str, page: u64, page_size: u64) ...
fn crate_reverse_dependencies_all(&self, _crate: &str) ...
```

Reply:
I like that, I'm working on this.